lib/features/ships/data/models/ships_model.dart

```diff
@@ -38,17 +38,17 @@ class ShipsModel {
       this.id});
 
   ShipsModel.fromJson(Map<String, dynamic> json) {
```

Comment:
please consider using 
https://pub.dev/packages/json_serializable

in models.